source/index.html.md

```diff
@@ -519,12 +564,12 @@ status | string | response status -->
   <meta charset="utf-8" />
 </head>
 <body>
-  <upward-link></upward-link>
-  <script src="https://api.upwardfi.com/upward-link.js"></script>
+  <highline-link></highline-link>
+  <script src="https://api.highline.co/highline-link.js"></script>
   <script type="text/javascript">
-  upwardLink.create({
+  highlineLink.create({
     plugin_key: 'your_plugin_key',
-    api_host: 'https://api-sandbox.upwardfi.com',
+    api_host: 'https://api-sandbox.highline.co',
```

Comment:
Can you work with @KurtBrowder. There are new APIs.
1. createPayrollDataConnection
2. createPayrollPaymentConnection